### PR TITLE
[APR] Add rewards for other pools on base and polygon zkevm

### PR DESCRIPTION
**Pull request by luizakp**
This PR:
- Refactor rewards calculation to follow [Balancer's calculation](https://github.com/balancer/balancer-sdk/blob/aa64364d00a21734b87e9b57ccaf6726ef3513e5/balancer-js/src/modules/pools/apr/apr.ts#L324)

https://www.loom.com/share/4b034a0614bb4b70abb712b89595ddc1?sid=93586b7a-efd5-4033-b9ec-693d824f2852

[TODO](https://linear.app/bleu-llc/issue/BAL-1135/altenative-for-token-price): find a way to get token historical prices to polygon zkevm. https://coins.llama.fi/prices/historical/1696896000/polygon-zkevm:0xa2036f0538221a77a3937f1379699f44945018d0?searchWidth=6h doesn't work

### CODE REVIEW

**Comment by ribeirojose** [comment]
> [TODO](https://linear.app/bleu-llc/issue/BAL-1135/altenative-for-token-price): find a way to get token historical prices to polygon zkevm. https://coins.llama.fi/prices/historical/1696896000/polygon-zkevm:0xa2036f0538221a77a3937f1379699f44945018d0?searchWidth=6h doesn't work

use polygon_zkevm instead: https://coins.llama.fi/prices/historical/1696896000/polygon_zkevm:0xa2036f0538221a77a3937f1379699f44945018d0?searchWidth=6h

how I found this out: https://github.com/DefiLlama/defillama-server/blob/master/coins/src/getChains.ts

File: apps/balancer-tools/pre-build.js [hunk 1]
```diff
@@ -97,6 +97,8 @@ const UPPER_CASE_TO_NETWORK = {
   OPTIMISM: "OPTIMISM",
   GNOSIS: "GNOSIS",
   ARBITRUM: "ARBITRUM",
+  BASE: "BASE",
+  AVALANCHE: "AVALANCHE",
```
Comment (ribeirojose) [inline apps/balancer-tools/pre-build.js:101]: 🙇  good catch!

**Review by ribeirojose — COMMENTED** [review]
LGTM, but defer to @mendesfabio

**Comment by luizakp** [comment]
image
working for zkvem now too

**Review by mendesfabio — DISMISSED** [review]

**Pull request merged**